assets/css/main.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .container {
-  @apply px-4;
+  @apply px-8;
```

Comment:
@lukicenturi 
I have been doing some testing and I think it would look better if we keep px-4 for mobile and do `md:px-24`

We also need to apply the tailwind utilities it seems for md to work.

Also the controls in the download page carousel seem to need adjustment too.

Reply:
I think md:px-24 is too big. I adjust it to md:px-12. Can you take a look again?